UM/PluginRegistry.py

```diff
@@ -31,6 +32,10 @@ def __init__(self):
         self._application = None
         self._active_plugins = []
 
+        preferences = Preferences.getInstance()
+        preferences.addPreference("general/disabled_plugins", "")
+        self._disabled_plugins = None
```

Comment:
It would have been nice if I had added a comment to that effect. I can do another PR for that, but that may just result in more overhead.